wp-content/plugins/jetpack/_inc/build/custom-post-types/comics/comics.min.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array(), 'version' => '4480d18b6ad25ddf9e5a620335fcf32c');
```

Comment:
:no_entry_sign: **Error( severity 5 )**: When a multi-item array uses associative keys, each value should start on a new line (*WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound*).